cornflow-server/cornflow/endpoints/login.py

```diff
@@ -55,10 +51,24 @@ def log_in(self, **kwargs):
         if auth_type == AUTH_DB:
             user = self.auth_db_authenticate(**kwargs)
             response.update({"change_password": check_last_password_change(user)})
+            current_app.logger.info(f"User {user.email} logged in successfully using database authentication")
```

Comment:
I prefer if we do not log the user email. Maybe better to just log the `user.id`?